src/plugins/base.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, bot):
         """
         self.bot = bot
 
-    def handle_tokens(self, channel, msg, keywords, callback):
```

Comment:
I understand that, but you can still have channel as the first parameter AND *args for the rest, I don't see why we need to move channel into args...